docs/docs/features/dongle.mdx

```diff
@@ -0,0 +1,208 @@
+---
+title: Keyboard Dongle
+sidebar_label: Keyboard Dongle
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+:::warning
+This implementation is experimental and subject to change.
+
+The information below is for [shield+board](../faq.md#why-boards-and-shields-why-not-just-keyboard) configuration, [board](../faq.md#what-is-a-board) builds is not documented.
+:::
+
+A bluetooth dongle can be added to any keyboard running ZMK. The result is a [split keyboard](./split-keyboards.md) with the dongle as "central". There are a number of advantages to adding a dongle, but also some disadvantages:
+
+Benefits:
+
+- The "central" dongle results in increased battery life for the keyboard/former "central", as it is now a peripheral.
+- It is easier to connect to the host device.
+
+Disadvantages:
+
+- An extra microcontroller is needed.
+- The keyboard becomes unusable without the dongle.
+
+:::note
+Depending on how the dongle is used, there are some additional [latency considerations](./split-keyboards.md#latency-considerations) to keep in mind. 
+:::
```

Comment:
I think it would be better to be specific rather than vague about this topic, as the latency situation can be confusing if you don't take your time to think about it.